server_admin/topics/sessions/preloading.adoc

```diff
@@ -0,0 +1,31 @@
+=== Offline Sessions Preloading
+
+Offline sessions besides the Infinispan caches are also stored in a database which means they will be available even after server restart. By default the offline sessions are preloaded from the database into the Infinispan caches during the server startup. However this approach has a drawback in case of there are many offline sessions to be preloaded. It can significantly slow down the server startup time.
```

Comment:
Change "Offline sessions besides the Infinispan caches are also stored"
TO
"in addition to Infinispan caches, offline sessions are stored..."
Add a comma after By default
Change "in case of" to "if"